### PR TITLE
augmented bash completion, fixed installed file permissions

### PR DESCRIPTION
**Pull request by rhaschke**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,11 +52,23 @@
...
     ['completion/catkin_tools-completion.bash']))
 
+
+class PermissiveInstall(install):
+    def run(self):
+        install.run(self)
+        if os.name == 'posix':
+            for file in self.get_outputs():
+                # all installed files should be readable for anybody
+                mode = ((os.stat(file)[ST_MODE]) | 0444) & 07777
```
Comment (dirk-thomas) [inline setup.py]: This line is not valid Python 3 syntax. You have to write octal number in the following style: `0o1234`

**Comment by wjwwood** [comment]
lgtm, but could you expand on what this didn't work for you before?

**Comment by rhaschke** [comment]
* The frequently used config options --whitelist, --blacklist, --install were not yet supported by bash completion.
* installing with `sudo python setup.py` left the installed files with read permission for root only on my Ubuntu 14.04

Both issues were fixed.

**Comment by wjwwood** [comment]
Interesting, ok thanks.

**Pull request merged**